tools/run_tests/run_tests.py

```diff
@@ -846,6 +846,7 @@ def _get_pythons(self, args):
             else:
                 return (
                     python27_config,
+                    python36_config,
```

Comment:
The concern here is how much longer are the tests going to take on kokoro. We are generally trying to reduce the test duration on PRs.  If the increase is reasonable and python jobs aren't taking too long, this is probably fine, but we should track how much of an increase adding python36 is.